src/codegeneration/ExplodeExpressionTransformer.js

```diff
@@ -750,6 +752,43 @@ export class ExplodeExpressionTransformer extends ParseTreeTransformer {
     return createCommaExpression(expressions);
   }
 
+  transformFunctionExpression(tree) {
+    // function () {}
+    // =>
+    // $0 = function () {}, $0
+    return this.createCommaExpressionBuilder_().build(tree);
+  }
+
+  transformArrowFunctionExpression(tree) {
+    // () => {}
+    // =>
+    // $0 = () => {}, $0
+    return this.createCommaExpressionBuilder_().build(tree);
```

Comment:
We do not want to descend into the arrow function. There is a test for this. My conclusing in that bug was wrong.